### ISSUE TITLE
Not very easy to use. 

### ISSUE THREAD

**Issue by AdamFenix** [ThauEx/ffrk-proxy#146]
I have spent a few hours trying to get this to work. Every time I try to set it up I get a bunch of syntax error in the CMD menu and when I try to use android (legacy) or proxydroid, nothing at all happens even when I edit the config to my liking. I am sure I am missing something but its still been a frustrating few hours. I reset my wifi and tried again but now I get a pop up saying it cant connect to the wifi.

**Comment by ThauEx** [comment]
Actually it is, the readme is not hard to follow and you only have to configure the proxy.   
You have to provide more details, otherwise I cannot help you.

**Comment by AdamFenix** [comment]
its been a few days and I still havent been able to get it to work. I finally got the certificate to install but still nothing.

**Comment by ThauEx** [comment]
I've been busy the last days...   
Try to get it working without config first. Phone configuration is basicly setting the url for proxy.pac in proxy settings and for iOS the root certificate. After that it should work.  
For custom config, create a new default.yml inside config and put your settings there.

**Comment by AdamFenix** [comment]
I understand and I appreciate you getting back to me but it just will not load up. I have tried now on both android, iOS, and emulators. I deleted everything and started again from a fresh download. I ran the .bat and edited my phones proxy setting and the game just hangs on the "A connection error has occurred. Retry?" I have attached photos showing just running without editing config and after renaming the default.whitelist to just default.

![first](https://user-images.githubusercontent.com/45781315/50079835-b099f200-01b8-11e9-91b8-63be92233004.png)
![second](https://user-images.githubusercontent.com/45781315/50079847-b55ea600-01b8-11e9-984d-e2555aa40b64.png)

**Comment by AdamFenix** [comment]
I understand and I appreciate you getting back to me but it just will not load up. I have tried now on both android, iOS, and emulators. I deleted everything and started again from a fresh download. I ran the .bat and edited my phones proxy setting and the game just hangs on the "A connection error has occurred. Retry?" I have attached photos showing just running without editing config and after renaming the default.whitelist to just default.
![first](https://user-images.githubusercontent.com/45781315/50080248-ad533600-01b9-11e9-9e42-9f37e257bbdf.png)
![second](https://user-images.githubusercontent.com/45781315/50080254-af1cf980-01b9-11e9-9703-d645546ecafc.png)

**Comment by ThauEx** [comment]
Hm... It fails to parse the config, I can check later if the same happens to me.

**Comment by AdamFenix** [comment]
I really appreciate the help

**Comment by ThauEx** [comment]
I can reproduce this issue, will try to make a fix for that.

**Comment by AdamFenix** [comment]
I hope its an easy one to fix

**Comment by ThauEx** [comment]
Replace `lib/config.js` with this one: https://raw.githubusercontent.com/ThauEx/ffrk-proxy/master/lib/config.js  
Issue is fixed in master branch, but no new release yet.

**Comment by AdamFenix** [comment]
Thanks, that seems to be working now.
